ADAL/src/cache/ADTokenCacheItem.m

```diff
@@ -30,13 +30,17 @@
 #import "ADTokenCacheItem+Internal.h"
 
 @implementation ADTokenCacheItem
+{
+    NSString *_storageAuthority;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_storageAuthority" to match the regular expression: ^[a-z][a-zA-Z0-9]*$ [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS116)